package.json

```diff
@@ -44,13 +44,29 @@
             {
                 "command": "aks.periscope",
                 "title": "Run AKS Periscope"
+            },
+            {
+                "command": "azure-deploy.ConfigureCICDPipeline",
+                "title": "Configure CI/CD"
+            },
+            {
+                "command": "azure-deploy.BrowseCICDPipeline",
+                "title": "Browse All Pipelines"
             }
         ],
         "menus": {
             "commandPalette": [
                 {
                     "command": "aks.selectSubscriptions",
                     "when": "never"
+                },
+                {
+                    "command": "azure-deploy.ConfigureCICDPipeline",
+                    "when": "never"
+                },
+                {
+                    "command": "azure-deploy.BrowseCICDPipeline",
```

Comment:
💡 Same here: camel casing please, for consistency with other commands in this extension.